### DISCUSSION TITLE
lsp: import all action

### DISCUSSION THREAD

**Discussion by DoctorRainer** [helix-editor/helix#9920] [General]
Like in WebStorm, not like in VS Code.

VS Code just picks first matching import whether WebStorm asks you for every conflicting import, for example we have Error in std and in serde

**Comment by pascalkuthe** [comment]
Code actions aren't controlled by the editor they are sent by the language server.

You need to raise this feature request with whatever language server you are using

**Comment by DoctorRainer** [comment]
@pascalkuthe weird, I have this option in VS Code for typescript
